templates/taxi/manufacturer_list.html

```diff
@@ -7,7 +8,10 @@ <h1>
       Create
     </a>
   </h1>
-
+  <form action="" method="get" class="form-inline">
+    {{ search_form|crispy }}
+    <input class="btn btn-secondary" type="submit" value="🔎" style="margin-left: 5px">
+  </form>
```

Comment:
Move this to a separate template and include where needed. Don't repeat yourself